File: uri/salario/salario.java
import java.util.*;
import java.io.IOException;

public class Main{

    public static double calcularsalario( int horas, double valor ){
        double foo;
        foo = (double)horas * valor;
        return (foo);
    }

    public static void main(String[] args) throws IOException {
        Locale.setDefault( Locale.US );
        Scanner input = new Scanner(System.in);
        int num, horas;
        double valor;

        num = input.nextInt();
        horas = input.nextInt();
        valor = input.nextDouble();
        System.out.printf("NUMBER = %d\n", num);
        System.out.printf("SALARY = U$ %.2f\n", calcularsalario(horas, valor));
        input.close();
    }
}

File: uri/salario/salario.kt
import java.util.Scanner
import kotlin.text.*

fun main(args: Array<String>){

    var input = Scanner(System.`in`) 

    var num : Int = input.nextInt()
    var horas : Int = input.nextInt()
    var valorhora : Double = input.nextDouble()

    var result : Double = (horas.toDouble()) * valorhora

    println ("NUMBER = $num")
    println ("SALARY = U$ %.2f".format(result))
}